runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -87,7 +87,7 @@ public IStorage ConfigureStorage(BotSettings settings)
 
         public bool IsSkill(BotSettings settings)
         {
-            return settings?.Skill?.IsSkill == true;
+            return settings?.SkillConfig?.IsSkill == true;
```

Comment:
Does this need to be 
```suggestion
            return settings?.SkillConfiguration?.IsSkill == true;
```